contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -73,7 +73,7 @@ typedef ResetConnectionData *ResetConnection;
 TdsRequestCtrlData *TdsRequestCtrl = NULL;
 
 ResetConnection resetCon = NULL;
-static bool resetTdsConnectionFlag = false;
+bool resetTdsConnectionFlag = false;
```

Comment:
Should we rather use `GetResetTDSConnectionFlag()` instead of making this global?